### PR TITLE
feat(explorer): use explored for host metrics

### PR DESCRIPTION
**Pull request by telestrial**

### CODE REVIEW

**Comment by telestrial** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/860?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#860** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/860?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/860?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#859** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/859?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#858** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/858?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: apps/explorer/app/page.tsx [hunk 1]
```diff
@@ -41,38 +42,51 @@ export default async function HomePage() {
...
   console.log(new Date().toISOString(), {
-    metrics: humanBytes(Buffer.byteLength(JSON.stringify(metrics || ''))),
     latestBlocks: humanBytes(
       Buffer.byteLength(JSON.stringify(latestBlocks || ''))
     ),
     exchangeRates: humanBytes(
       Buffer.byteLength(JSON.stringify(exchangeRates || ''))
     ),
     hosts: humanBytes(Buffer.byteLength(JSON.stringify(hosts || ''))),
+    hostMetrics: humanBytes(
```
Comment (alexfreska) [inline apps/explorer/app/page.tsx:74]: This byte size logging code was from debugging a past issue that I thought might be due to the payload size. If you find it useful we can leave it, but feel free to remove it.

File: apps/explorer/components/Home/index.tsx [hunk 2]
```diff
@@ -150,13 +147,13 @@ export function Home({
...
                   {getDownloadCost({
-                    price: metrics?.average.settings.download_price,
+                    price: metrics?.priceTable.downloadbandwidthcost,
                     exchange,
                   })}
                 </Text>
                 <Text color="subtle">
                   {getDownloadCost({
-                    price: metrics?.average.settings.download_price,
+                    price: metrics?.priceTable.downloadbandwidthcost,
```
Comment (alexfreska) [inline apps/explorer/components/Home/index.tsx:156]: Not related to this PR but question: Are all these metrics fields averages? Do we plan to support things besides average now or in the future? If so should we put these under an averages key?

**Review by alexfreska — APPROVED** [review]
Two non-blocking comments.

File: apps/explorer/app/page.tsx (hunk 1 above)
Comment (telestrial) [reply apps/explorer/app/page.tsx:74]: Removed!

File: apps/explorer/components/Home/index.tsx (hunk 2 above)
Comment (telestrial) [reply apps/explorer/components/Home/index.tsx:156]: They are averages, yes. @chris124567 may have an answer here. I assume you mean like min and max?

**Comment by telestrial** [comment]
### Merge activity

* **Dec 16, 2:43 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/SiaFoundation/web/860).

**Pull request merged**